jep/README.md

```diff
@@ -0,0 +1,202 @@
+---
+title: React.js at Jupyter
+authors: Eric Charles (@echarles)
+issue-number: <pre-proposal-issue-number>
+pr-number: <proposal-pull-request-number>
+date-started: 2022-02-08
+---
+
+> JEP for React.js at Jupyter
```

Comment:
We can do that. The word "UI" tells well what it does. Component can be anything like python libraries... Maybe "UI Components" instead of "UI Toolkit"

Reply:
Indeed UI clarifies the scope and I like _UI components_ that also improves the scope definition.

Reply:
Based on the content of the JEP below a different name than `React.js` would indeed be preferable.